guide/resources/design-research.md

```diff
@@ -97,6 +97,12 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [The State of Lightning Volume 2](https://arcane.no/research/reports/the-state-of-lightning-volume-2)
+
+_2022 by Arcane research_
```

Comment:
```suggestion
_Login required to access. 2022 by Arcane research_
```

Reply:
@paulosacramento you'll have to press `Commit suggestion` above if you want to accept the change. `Resolve conversation` only closes the conversation without accepting the change.